web-components/src/components/advance-list/AdvanceList.test.ts

```diff
@@ -272,15 +272,6 @@ describe("advanceList Component", () => {
     });
 
     describe("Accessibility and Error Handling", () => {
-      test("should show loading spinner when isLoading is true", async () => {
```

Comment:
Dont we need this test case?

Reply:
we are removing this because the loader is being handled from client side